src/common/middlewares/auth.middleware.ts

```diff
@@ -73,12 +73,14 @@ export class AuthMiddleware implements NestMiddleware {
         }
 
         req.mw_userid = user.id;
+        next();
+
       } catch (error) {
         console.error('Something went wrong:', error);
         req.mw_userid = '';
+        next(new UnauthorizedException('Invalid token'));
```

Comment:
will this apply to save application? will token be required?

Reply:
no